docs/02_getting-started/01_try-antelope-in-dune.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Try Antelope in DUNE"
+title: "Try EOS in DUNE"
```

Comment:
Shouldn't we also update the filename `try-antelope`... --> `try-eos`... since we changed the title to "Try EOS..."?